float64 xpos
float64 ypos
float64 spacing_first
float64 spacing
float64 inclination
int32 rungs
float64 legs_width
float64 rail_height
float64 walkway_rail_height
float64 direction